vdev/src/testing/integration.rs

```diff
@@ -145,13 +167,17 @@ struct Compose {
 
 impl Compose {
     fn new(test_dir: PathBuf, env: Environment, network: Option<String>) -> Result<Option<Self>> {
-        let path: PathBuf = [&test_dir, Path::new("compose.yaml")].iter().collect();
+        let path: PathBuf = [&test_dir, Path::new("compose-temp.yaml")].iter().collect();
         match path.try_exists() {
             Err(error) => Err(error).with_context(|| format!("Could not lookup {path:?}")),
             Ok(false) => Ok(None),
             Ok(true) => {
                 #[cfg(unix)]
                 let config = ComposeConfig::parse(&path)?;
+                println!(
```

Comment:
I will remove this once comes out of draft, but figured this might be helpful for those trying to test the changes out themselves